eitprocessing/parameters/tidal_impedance_variation.py

```diff
@@ -0,0 +1,205 @@
+from dataclasses import dataclass, field
+from functools import singledispatchmethod
+from typing import Literal
+
+import numpy as np
+
+from eitprocessing.datahandling.breath import Breath
+from eitprocessing.datahandling.continuousdata import ContinuousData
+from eitprocessing.datahandling.eitdata import EITData
+from eitprocessing.datahandling.intervaldata import IntervalData
+from eitprocessing.datahandling.sequence import Sequence
+from eitprocessing.features.breath_detection import BreathDetection
+from eitprocessing.features.pixel_inflation import PixelInflation
+from eitprocessing.parameters import ParameterCalculation
+
+
+@dataclass
+class TIV(ParameterCalculation):
+    """Compute the tidal impedance variation (TIV) per breath."""
```

Comment:
When we first started working on pixel inflations we discussed that we shouldn't call it breaths since pixels do not have independent breaths but are always associated with breaths in the continuous data. But I agree that it can cause confusion. Would you suggest changing it to pixel breaths or rather explaining the nomenclature and keeping it consistent?

Reply:
I still think breath is not the best name, but then again, neither is inflation, since impedance changes are not always directly due to inflation, but can be due to e.g. movement. Maybe it's easiest to just use breath everywhere, and make a small footnote that impedance changes are not necessarily just due to inflation.

Reply:
I agree, for consistency I have changed it to pixel breaths.